src/Duo.Clients.Wpf/Presentation/ManutenzioneTrattamentoViewModel.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Duo.Domain.ViewModels.Trattamenti;
+using Duo.Messages.Trattamenti.Commands;
+using Radical.CQRS.Client;
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+using Topics.Radical.ComponentModel.Messaging;
+using Topics.Radical.Windows.Presentation;
+using Topics.Radical.Windows.Presentation.ComponentModel;
+using Topics.Radical.Windows.Presentation.Messaging;
+using Topics.Radical.Windows.Presentation.Services.Validation;
+
+namespace Duo.Clients.Wpf.Presentation
+{
+    class ManutenzioneTrattamentoViewModel : AbstractViewModel, IRequireValidation
+    {
+        readonly IMessageBroker broker;
+        readonly Services.AppSettings settings;        
+
+        public Guid Id
```

Comment:
`public Guid Id` non ti serve che:

* sia una Radical property
* che sia public
* che sia in get/set

probabilmente basterebbe che sia un normalissimo field